test/Microsoft.AspNetCore.Mvc.Core.Test/Authorization/AuthorizeFilterTest.cs

```diff
@@ -104,7 +104,7 @@ public void InvalidUser()
         {
             // Arrange
             var authorizeFilter = new AuthorizeFilter(new AuthorizationPolicyBuilder().RequireAssertion(_ => true).Build());
-            var authorizationContext = GetAuthorizationContext(services => services.AddAuthorization(), anonymous: true);
```

Comment:
If the behavior here is external to MVC do we need this many tests? How many of these are just retesting policyevaluator?